pkgs/development/python-modules/fido2/default.nix

```diff
@@ -9,17 +9,27 @@
 
 buildPythonPackage rec {
   pname = "fido2";
-  version = "0.8.1";
+  version = "0.9.1";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1hzprnd407g2xh9kyv8j8pq949hwr1snmg3fp65pqfbghzv6i424";
+    hash = "sha256-hoDuJSOOIwdZbrOQCg+MDZzJEYkUbtgDlUTxo6ad/m4=";
   };
 
   propagatedBuildInputs = [ six cryptography ];
 
   checkInputs = [ mock pyfakefs ];
```

Comment:
Maybe try using ``checkInputs = [ pytestCheckHook ... ];``? You might not need the custom checkPhase if that works. Pytest should nominally run all ``unittest`` tests out-of-the-box.

Reply:
Unfortunately, that approach doesn't seem to work.